src/main/resources/init_db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE `taxi_db`;
+
+CREATE TABLE `manufacturers` (
+                                 `id` bigint NOT NULL AUTO_INCREMENT,
+                                 `name` varchar(255) DEFAULT NULL,
+                                 `country` varchar(255) DEFAULT NULL,
+                                 `is_deleted` tinyint NOT NULL DEFAULT '0',
+                                 PRIMARY KEY (`id`)
```

Comment:
Primary key constraint could be added immediately after column declaration 
```
`id` bigint AUTO_INCREMENT PRIMARY KEY,
```
and in this case NOT_NULL constraint is redundant, as PRIMARY KEY already consist of NOT_NULL and UNIQUE constraints